tests/test_sanity.py

```diff
@@ -241,3 +243,33 @@ def test_from_container_spawned_process(
         assert_collapsed(collapsed)
 
     assert b"Profiling spawned process" in container.logs()
+
+
+@pytest.mark.parametrize("in_container", [True])
+@pytest.mark.parametrize("profiler_type", ["attach-maps"])
+@pytest.mark.parametrize("runtime", ["nodejs"])
+@pytest.mark.parametrize("application_image_tag", ["without-flags"])
+def test_container_name_when_stopped(
+    docker_client: DockerClient,
+    gprofiler_docker_image: Image,
+    output_directory: Path,
+    output_collapsed: Path,
+    runtime_specific_args: List[str],
+    profiler_type: str,
+    profiler_flags: List[str],
+    application_docker_container: Container,
+) -> None:
+    profiler_flags.extend(["-d", "20"])
+    container = start_gprofiler_in_container_for_one_session(
+        docker_client, gprofiler_docker_image, output_directory, output_collapsed, runtime_specific_args, profiler_flags
+    )
+    try:
+        wait_event(20, Event(), lambda: re.search(rb"Started perf", container.logs()) is not None)
```

Comment:
After you cange to Java you can fix this log searching for the log message of profiling a java process.